aexpect/remote.py

```diff
@@ -344,7 +344,8 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                  interface=None, identity_file=None,
                  status_test_command="echo $?", verbose=False, bind_ip=None,
                  preferred_authenticaton='password',
-                 user_known_hosts_file='/dev/null'):
+                 user_known_hosts_file='/dev/null',
+                 extra_params=[]):
```

Comment:
This is not safe due to known python problems like https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument. I would recommend to set `extra_params` to `None` and then in the function body to do something standard like
```python
extra_params = [] if extra_params is not None else extra_params
```

Reply:
Yes please, alternatively the `extra_params` could be just a simple string as, in the end, it's merged into a single string anyway. Would be IMO easier for the users, but it's not a strong opinion.

Reply:
Yes, this could be even easier to implement and could require some renaming of `extra_params` to something like `extra_cmdline`, `extra_args_str`, etc.

Reply:
Updated per your suggestions. Now, it si extra_cmdline variable which is stored internally to extra_params list if exists as well as verbose parameter. I put both strings to list, so I can create a final string including spaces pythonic way using join function. The other way would be to have multiple if statements checking if verbose or extra_cmdline string is defined and put the space between those if they are both defined. But that would not be a "nice" code.